codespell_lib/data/dictionary.txt

```diff
@@ -12306,6 +12309,7 @@ faught->fought
 fauilures->failures
 faund->found
 favoutrable->favourable
+favuourites->favourites
```

Comment:
Add the US version as a secondary maybe.

Reply:
done. and added it to enGB_to_enUS